modules/cudf/src/series.ts

```diff
@@ -650,6 +650,25 @@ export class AbstractSeries<T extends DataType = any> {
     return this.__construct(this._col.gather(selection._col));
   }
 
+  /**
+   * Return a copy of this Series.
+   *
+   *
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   *
+   * const a = Series.new(["foo", "bar", "test"]);
+   *
+   * a.copy() // StringSeries ["foo", "bar", "test"]
+   * ```
+   */
+  copy(memoryResource?: MemoryResource): Series<T> {
+    // const lhs = <Column>(this._col);
+    return Series.new(this._col.copy(memoryResource));
+  }
+  // this.__construct(this._col.copy()); }
```

Comment:
Are these comments needed?
`// const lhs = <Column>(this._col);` - line 667

`// this.__construct(this._col.copy()); }` - line 670